src/components/Common/OrganizationCard.js

```diff
@@ -106,96 +106,94 @@ class OrganizationCard extends Component {
             encodedCoordinates;
 
         return (
-            <div ref="cardRef">
```

Comment:
this extra div doesn't appear to be necessary, and it's best to not have extra divs if they can be avoided